5.6/Dockerfile.rhel7

```diff
@@ -0,0 +1,61 @@
+FROM openshift3/sti-base
```

Comment:
Sorry, but I'm totally lost in this base image nightmare.

openshift3/sti-base is used in other collections and build in OSBS

I have absolutely no idea of diff between openshift3/sti-base and openshift/base-rhel7. So I really need clear explanation, expectation, description (link to base image Dockerfile to understand), etc

In all case, I won't touch this before september (hollidays)